app/services/__init__.py

```
@@ -16,6 +16,8 @@
 # from .farmer_service import FarmerService
 # from .product_service import ProductService
 
+from .user_service import *
```

Ruff (0.11.9)

`from .user_service import *` used; unable to detect undefined names

(F403)